### DISCUSSION TITLE
Breakaway Project Files?

### DISCUSSION THREAD

**Discussion by steveulrich** [o3de/rfcs#34] [General]
With Lumberyard going open source, will the Breakaway Project Files be made available to the public?

**Comment by amzn-phist** [comment]
As cool as that sounds, there are some problems with doing this:
1. All the code and assets for Breakaway are still owned by Amazon, and not only that, Games is a separate part of the organization than the Lumberyard/O3DE engine team.
2. The engine used for Breakaway was a very early version of Lumberyard, so it was mostly CryEngine.  There would be too many technical issues getting that project working in O3DE.
